### DISCUSSION TITLE
interact with postgres tables from non-public schemas

### DISCUSSION THREAD

**Discussion by kmdelucaNG** [Budibase/budibase#3124] [Help]
New to budibase.  I have v0.9.168.

Saw this PR for enabling the loading of postgres tables from non-public schemas:
https://github.com/Budibase/budibase/pull/2716

Works good - I can now see both the public and non-public schemas/tables.
However, it looks like the schema namespace is not retained within budibase for the non-public tables. Any actions performed on them are applied to the default/public schema and thus fail.  Is there a way to resolve that for a mix of public and non-public tables without having to write custom queries for everything?
Thanks.

**Comment by mjashanks** [comment]
Hi There -we have an open issue for this, which we will look to address next month: https://github.com/Budibase/budibase/issues/3039

Custom queries are the only answer for now.

**Reply by kmdelucaNG** [reply]
sounds good, thanks for the response.
